initrd/bin/mount-usb

```diff
@@ -126,7 +126,7 @@ fi
 #Check if $USB_MOUNT_DEVICE is a LUKS encrypted device.Ff so, luksOpen it and replace $USB_MOUNT_DEVICE with /dev/mapper/usbdev
 is_Luks_luksOpen
 if [ "$1" = "rw" ]; then
-  mount -o rw $USB_MOUNT_DEVICE /media
+  mount -o rw $USB_MOUNT_DEVICE /media || die "Unable to remount /media into Read Write mode. Is the device write protected?" 
 else
-  mount -o ro $USB_MOUNT_DEVICE /media
+  mount -o ro $USB_MOUNT_DEVICE /media || || die "Unable to mount $USB_MOUNT_DEVICE."
```

Comment:
Typo with this double ||.

Also, do we really need to die here if one can't mount the USB? I'd rather show errors to the user and move on than drop to recovery shells.

Reply:
I'll do a warn then.

_Sent from my Galaxy S3 using [FastHub-Libre](https://github.com/thermatk/FastHub-Libre/)_

Reply:
Removed.

Reply:
> Typo with this double ||.
> 
> Also, do we really need to die here if one can't mount the USB? I'd rather show errors to the user and move on than drop to recovery shells.

@kylerankin You're right. Removed what was not usb related. OEM can launch docking station IDE device by calling "mount /media" anyway. Let's not mix things.